packages/shared/package.json

```
@@ -33,11 +33,15 @@
     "@trpc/server": "^11.5.1",
     "adm-zip": "^0.5.16",
     "axios": "^1.13.1",
+    "nanoid": "^5.1.5",
```

Biome (2.1.2)

[error] The key nanoid was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)